iocore/net/P_SSLConfig.h

```diff
@@ -161,6 +159,25 @@ struct SSLCertificateConfig {
   static int configid;
 };
 
+struct SSLTicketParams : public ConfigInfo {
+  ssl_ticket_key_block *default_global_keyblock;
+  char *ticket_key_filename;
+  void LoadTicket();
+  void cleanup();
+  ~SSLTicketParams();
+};
+
+struct SSLTicketKeyConfig {
+  static void startup();
+  static bool reconfigure();
+  static SSLTicketParams *acquire();
```

Comment:
As commented below, move the simple implementations up here, for automatic inlinining.